### PR TITLE
Clarify client verification key with examples.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@mender.io>

### CODE REVIEW

**Comment by drewmoseley** [comment]
Acked-by: Drew Moseley \<drew.moseley@mender.io\>

@estenberg any comments before I merge?

**Comment by drewmoseley** [comment]
@kacf does this need to go into branch 1.1.x as well?

**Comment by kacf** [comment]
> does this need to go into branch 1.1.x as well?

Yes.

**Comment by drewmoseley** [comment]
@kacf any idea why this is now showing conflicts?  I don't see anything obvious in my local checkout.

File: 04.Artifacts/06.Building-for-production/docs.md [hunk 1]
```diff
@@ -66,5 +66,13 @@ Note in particular the `:` after the directory; this is mandatory.
...
+The best way to include a public verification key in the client is to add it to your own layer. Set the name of the verification key to `artifact-verify-key.pem` and append it to `SRC_URI` of the `mender` application before building the Yocto client image. For example:
 
+```bash
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI_append = " file://artifact-verify-key.pem"
+```
+
+Note that it is also possible (but not recommended) to use `local.conf`, by using [the same method as for client certificates](#using-localconf), adding `pn-mender` to the variable names.
+
+For more information about some alternate approaches please follow [MENDER_ARTIFACT_VERIFY_KEY documentation](../Variables#mender-artifact-verify-key)).
```
Comment (estenberg) [inline 04.Artifacts/06.Building-for-production/docs.md]: I think this link will be broken when deployed to staging/prod. For some reason staging/prod only works with all-lowercase links, (e.g. ../variables#...) while the local env only works with uppercase. Should test in staging to make sure.

**Comment by estenberg** [comment]
Otherwise, great!

**Comment by estenberg** [comment]
Manually comitted in https://github.com/mendersoftware/mender-docs/commit/5696e62d29cd44da37f81714b222756cb527bca3

**Pull request closed without merging**